casepro/utils/__init__.py

```diff
@@ -59,6 +60,23 @@ def json_encode(data):
     return json.dumps(data, cls=JSONEncoder)
 
 
+def json_decode(data):
+    """
+    Decodes the given JSON as primitives
+    """
+    if isinstance(data, six.binary_type):
+        data = data.decode('utf-8')
```

Comment:
There are two places this is used - 1) handling requests made from Angular services and 2) handling responses/requests in the Junebug backend. I'm pretty certain 1) will always use UTF8. For 2) we need to hear from @rudigiesler @jpaidoussi et al.